source/includes/minio-mc-installation.rst

```diff
@@ -51,6 +51,13 @@
 
          mc --help
 
+      .. admonition:: Install from the MinIO Download Page
+         :class: note
+
+         To ensure you install the correct binary, download ``mc`` from the `MinIO Download Page <https://dl.min.io/client/mc/release/>`__.
+         Other repositories, including those used by common Linux package managers, may have other software with a similar name.
```

Comment:
So I *think* we can safely go farther here.

I am actually seeing `minio-client` in the arch/extra repos. Nothing in the ubuntu repos (thankfully?), and I can't be damned to figure our RHEL right now.

That  said, spitballing:

```suggestion
         MinIO does not officially publish it's binaries to common Linux repositories or package managers (Ubuntu, RHEL, Archlinux/AUR).
         The only official source of MinIO binaries is the <download page URL>.
         
         If installing the MinIO Client through your package manager, take extra steps to ensure you download the correct package.
         For example, some operating systems will install Midnight Commander ``mc`` by default, or will rename the MinIO client to ``mcli``.
         
         All documentation assumes the installation of the *official* ``mc`` client binary through the download page *only* with no changes to binary naming.
```

Reply:
I did this as a suggestion and did it wrong :face_with_spiral_eyes: